config/core/iptables-services.pan

```diff
@@ -0,0 +1,22 @@
+unique template config/core/iptables-services;
+
+# Install package providing iptables and ip6tables services
+'/software/packages' = {
```

Comment:
why is this not a depency of the component?
also, this is just `'/software/packages/{iptables-services}' = dict();`?